entity-framework/core/modeling/owned-entities.md

```diff
@@ -39,25 +39,39 @@ See the [full sample project](https://github.com/aspnet/EntityFramework.Docs/tre
 
 Owned types configured with `OwnsOne` or discovered through a reference navigation always have a one-to-one relationship with the owner, therefore they don't need their own key values as the foreign key values are unique. In the previous example, the `StreetAddress` type does not need to define a key property.  
 
-In order to understand how EF Core tracks these objects, it is useful to think that a primary key is created as a [shadow property](xref:core/modeling/shadow-properties) for the owned type. The value of the key of an instance of the owned type will be the same as the value of the key of the owner instance.
+In order to understand how EF Core tracks these objects, it is useful to know that a primary key is created as a [shadow property](xref:core/modeling/shadow-properties) for the owned type. The value of the key of an instance of the owned type will be the same as the value of the key of the owner instance.
 
 ## Collections of owned types
 
->[!NOTE]
+> [!NOTE]
 > This feature is new in EF Core 2.2.
 
-To configure a collection of owned types `OwnsMany` should be used in `OnModelCreating`. However the primary key will not be configured by convention, so it needs to be specified explicitly. It is common to use a complex key for these type of entities incorporating the foreign key to the owner and an additional unique property that can also be in shadow state:
+To configure a collection of owned types `OwnsMany` should be used in `OnModelCreating`.
+
+As any entity type the owned types need a primary key. If there are no good candidates on the CLR type EF Core will create one, but since these types are defined through a collection the foreign key is not unique, so we can't use just it as the primary key as we do for `OwnsOne`.
+
+The two most straightforward solutions are:
+- Using a single non-FK property. The contained values would need to be unique across all owners (e.g. if Parent 1 has Child 1, then Parent 2 cannot have Child 1), so the value doesn't have any inherent meaning. Since the FK is not part of the PK its values can be changed, so you could move a child from one parent to another one, however this usually goes against aggregate semantics.
```

Comment:
```suggestion
- Defining a surrogate primary key on a new property independent of the foreign key that points to the owner. The contained values would need to be unique across all owners (e.g. if Parent 1 has a Child 1, then Parent 2 cannot have a Child 1), so the value doesn't have any inherent meaning. Since the foreign key is not part of the primary key its values can be changed, so you could move a child from one parent to another one, however this usually goes against aggregate semantics.
```

Reply:
Perhaps we should say something in the intro to the topic of owned entities about the fact that a goal of ownership is to help model DDD aggregates, with some stable external link to the topic, for example https://martinfowler.com/bliki/DDD_Aggregate.html. Otherwise when we start mentioning them here it feels completely arbitrary.